add: detective helmet + a little rearrangement in his cabinet

WolfLox opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание
Добавляет в шкаф к детективу в мешок для одежды шлем детектива
Также перестановки в рамках кабинета для меньшей захламлённости кабинета. Все предметы которые должны быть в поясе теперь лежат в поясе. Револьвер получил собственную коробку в которой лежит вместе с патронами и кобурой.
<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->

## Ссылка на предложение/Причина создания ПР
https://discord.com/channels/617003227182792704/755125334097133628/1235623193411321907
<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

## Демонстрация изменений
![image](https://github.com/ss220-space/Paradise/assets/108519426/ede896b2-ee5f-4bc9-9947-8112cc86829b)
![image](https://github.com/ss220-space/Paradise/assets/108519426/bd32f89b-3092-4cbe-a8e5-a828eb90cf29)
![image](https://github.com/ss220-space/Paradise/assets/108519426/2e84d277-7006-464c-a715-746ab37b0d95)
![image](https://github.com/ss220-space/Paradise/assets/108519426/d6087175-0166-461c-8d9e-c1a999856cd3)
<!-- Здесь вы можете показать изменения внешне, к примеру новые спрайты, звуки или изменения карты. Или написать, что именно изменилось для игроков. Этот пункт полностью опционален и его можно удалить. -->

WolfLox committed:
dec helmet

Немного передвинул некоторые вещи из ящика дека раскидав их по его офису, чтобы не перегружать его шкаф.

WolfLox committed:
add revolver box

Просто коробка для револьвера детектива чтобы уменьшить захламление ящика.

Gottfrei commented on code/modules/clothing/head/helmet.dm:
```diff
@@ -35,6 +35,25 @@
+	desc = "Standard Security gear in olive color."
+	icon_state = "dechelmet"
+	item_state = "dechelmet"
+	flags_inv = HIDEHEADSETS
```
```suggestion
	flags_inv = HIDEHEADSETS|HIDEHEADHAIR
```

WolfLox replied:
Оно наследуется от обычного шлема который данный слот не закрывал, в ТЗ не было прописано покрытие волос шлемом так что как мне кажется не ошибка.

Gottfrei replied:
это не ошибка, а буквально проблемы визуала большей части шлемов. впрочем если не хочешь менять - дело твоё.

aeternaclose commented:
ждём всем селом нового коммита

WolfLox committed:
commit

только бы не сломалось только бы не сломалось только бы не сломалось только бы не сломалось
